README.md

```diff
@@ -97,6 +97,8 @@ To declare an API method, call `api.declare(options, handler)` with the followin
  * `scopes` - scopes required for this API endpoint, in disjunctive normal form (see below)
  * `deferAuth` - if true, authentication will not be checked automatically before the handler
    is invoked.  In this case, the handler must call `req.satisfies()`; see below.
+ * `noAuth` - if true, this endpoint will not be authenticated and clients should not send
```

Comment:
How about `public: true`?

can you explain a bit more why this is necessary?
`deferAuth` should generally be enough... since there is no reason to do auth until `satisfies()` is called..
hmm... I guess the problem is that `satisfies()` is synchronous?